server/src/agents/executor/toolkit.ts

```diff
@@ -217,33 +219,100 @@ export const getExecutorToolkit = (account: Account) => {
           return `Transaction not found for task: "${task}" [id: ${taskId}].`;
         }
 
-        const walletClient = createWalletClient({
-          account,
-          chain: getChain(parseInt(env.CHAIN_ID)),
-          transport: http(),
+        const chain = getChain(parseInt(env.CHAIN_ID));
+        if (!chain) return `Chain not supported`;
+
+        // const walletClient = createWalletClient({
```

Comment:
the wallet client doesnt need to be removed , the safe feature should be made optional and not mandatory